src/main/kotlin/me/owdding/skyocean/features/mining/scathas/Scathas.kt

```diff
@@ -0,0 +1,133 @@
+package me.owdding.skyocean.features.mining.scathas
+
+import me.owdding.ktmodules.Module
+import me.owdding.skyocean.config.features.mining.ScathaConfig
+import me.owdding.skyocean.utils.chat.ChatUtils
+import me.owdding.skyocean.utils.chat.ChatUtils.sendWithPrefix
+import tech.thatgravyboat.skyblockapi.api.events.base.Subscription
+import tech.thatgravyboat.skyblockapi.api.events.base.predicates.OnlyIn
+import tech.thatgravyboat.skyblockapi.api.events.base.predicates.TimePassed
+import tech.thatgravyboat.skyblockapi.api.events.entity.EntityInfoLineEvent
+import tech.thatgravyboat.skyblockapi.api.events.time.TickEvent
+import tech.thatgravyboat.skyblockapi.api.location.SkyBlockIsland
+import tech.thatgravyboat.skyblockapi.helpers.McClient
+import tech.thatgravyboat.skyblockapi.utils.extentions.cleanName
+import net.minecraft.sounds.SoundEvents
+import tech.thatgravyboat.skyblockapi.api.data.SkyBlockRarity
+import tech.thatgravyboat.skyblockapi.api.events.chat.ChatReceivedEvent
+import tech.thatgravyboat.skyblockapi.utils.text.Text
+import tech.thatgravyboat.skyblockapi.utils.text.TextBuilder.append
+import tech.thatgravyboat.skyblockapi.utils.text.TextColor
+import tech.thatgravyboat.skyblockapi.utils.text.TextStyle.bold
+import tech.thatgravyboat.skyblockapi.utils.text.TextStyle.color
+
+
+@Module
+object Scathas {
+
+    var worm: SpawnedWorm = SpawnedWorm(null)
+    var cooldown: Boolean = false
+
+    @Subscription
+    @OnlyIn(SkyBlockIsland.CRYSTAL_HOLLOWS)
+    fun getWorm(event: EntityInfoLineEvent) {
+        if (!ScathaConfig.wormAnnouncer) { return }
```

Comment:
```suggestion
        if (!ScathaConfig.wormAnnouncer) return
```

this is a bit cleaner, also applies to the rest of the instances where you early return